src/connectionController.ts

```diff
@@ -743,7 +743,11 @@ export default class ConnectionController {
   async onRemoveMongoDBConnection(): Promise<boolean> {
     log.info('mdb.removeConnection command called');
 
-    const connectionIds = Object.keys(this._connections);
+    const connectionIds = Object.entries(this._connections).filter(
+      ([, connection]) =>
```

Comment:
[nit] consider mapping this tuple to an object with id and connection fields - that way it'll be easier to know what you're destructuring into without having to remember what value is at what index.

Reply:
Saw that you're out today, so I went ahead and applied this suggestion - happy to revert it if you disagree with the change, but wanted to merge the PR to get into the next release